tool/gravity/cli/system.go

```diff
@@ -558,6 +559,68 @@ func systemServiceStatus(env *localenv.LocalEnvironment, pkg loc.Locator, servic
 	return nil
 }
 
+const systemStopMessage = `WARNING: This action will stop all Gravity and Kubernetes services on the node.
```

Comment:
Do we want to display warning banners for these commands? Could corresponding high-level commands do this automatically?